modules/addons/darkblend/css/dark-blend.css

```diff
@@ -581,7 +595,9 @@ ul.client-tabs>li>a:hover {
     border-color: #4e4e4e #4e4e4e var(--color-dark-5) #4e4e4e !important;
 }
 
-
+.navigation .navbar-collapse {
+    background-color: #232323 !important;
```

Comment:
Thank you for contributing.
I think it would be better if we change the values of the color `#232323` to the universal variable of `--color-dark-3`